extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopgate-project/doofinder-search",
```

Comment:
Ext name

Reply:
Could you check `devDeps` and `scripts` section.
We need to keep extension and deps as thin as needed
Every `dep` give you overhead with locking, updating, auditing, etc